src/settings-ui/Settings.UI/SettingsXAML/Views/PowerLauncherPage.xaml

```diff
@@ -477,6 +484,9 @@
             <controls:PageLink
                 x:Uid="LearnMore_Run"
                 Link="https://aka.ms/PowerToysOverview_PowerToysRun" />
+            <controls:PageLink
+                x:Uid="Run_FindMorePlugins"
+                Link="https://aka.ms/powerToysRunPlugins" />
```

Comment:
Link needs to be created

Reply:
You want to direct it to https://github.com/microsoft/PowerToys/blob/main/doc/thirdPartyRunPlugins.md , which would be live after merging, is that right?

Reply:
Yes!

Reply:
@crutkas , can we please create this alias?
"https://aka.ms/powerToysRunPlugins" to "https://github.com/microsoft/PowerToys/blob/main/doc/thirdPartyRunPlugins.md" ?

Reply:
https://aka.ms/powerToysRunPlugins is live ;)